README.md

```diff
@@ -27,8 +37,290 @@ pip install -e .
 
 ## What can AMICAL do for you ?
 
-See [example_NIRISS.py](example_NIRISS.py).
+AMICAL has been developed to provide an easy-to-use solution to process
+**A**perture **M**asking **I**nterferometry (AMI) data from major existing
+facilities:
+[NIRISS](https://jwst-docs.stsci.edu/near-infrared-imager-and-slitless-spectrograph)
+on the JWST (first scientific interferometer operating in space),
+[SPHERE](https://www.eso.org/sci/facilities/paranal/instruments/sphere.html) and
+[VISIR](https://www.eso.org/sci/facilities/paranal/instruments/visir.html) from
+the European Very Large Telescope (VLT) and
+[VAMPIRES](https://www.naoj.org/Projects/SCEXAO/scexaoWEB/030openuse.web/040vampires.web/indexm.html)
+from SUBARU telescope (and more to come).
+
+We focused our efforts to propose a user-friendly interface, though different
+sub-classes allowing to (1) **Clean** the reduced datacube from the standard
+instrument pipelines, (2) **Extract** the interferometrical quantities
+(visibilities and closure phases) using a Fourier sampling approach and (3)
+**Calibrate** those quantities to remove the instrumental biases.
+
+Looking for a quickstart into AMICAL? Just have a look to the example scripts
+made for [NIRISS](example_NIRISS.py) and [SPHERE](example_NIRISS.py).
+
+In addition (4), we include two external packages called
+[CANDID](https://github.com/amerand/CANDID) and
+[Pymask](https://github.com/AnthonyCheetham/pymask) to **analyse** the final
+outputs obtained from a binary-like sources (star-star or star-planet). We
+interfaced these stand-alone packages with AMICAL to quickly estimate our
+scientific results (e.g.: separation, position angle, contrast ratio, contrast
+limits, etc.) using different approaches (chi2 grid, MCMC, see
+[example_analysis.py](example_analysis.py) for details).
+
+## Tutorial
+
+In this tutorial, we will go through the different possibilities of AMICAL. You
+can find a detailed description of the principal functions and associated
+parameters in their docstrings (easily accessible with good editors, e.g.:
+vscode).
```

Comment:
You don't need to adverstise how to read docstrings. In fact, docstrings are meant to be accessible from outside text editors. The basic Python REPL offers a `help()` function to inspect docstrings in an interactive session.
IPython/Jupyter supports the `?` suffix that does the same thing.
Point is: if you think your users need to open the source code to understand how to use it, you're in trouble. The good news is, if you already have docstrings, it's (maybe) not the case.